service/src/main/java/uk/gov/hmcts/reform/fpl/model/event/ManageOrdersEventData.java

```diff
@@ -40,6 +42,9 @@
     String manageOrdersDirections;
     EPOType manageOrdersEpoType;
     String manageOrdersIncludePhrase;
+    List<C43OrderType> manageOrdersC43Orders;
+    String manageOrdersRecitalsAndPreambles;
+    String manageOrdersC43Directions;
```

Comment:
@SooryaSu 
Could you elaborate to what made you create `manageOrdersC43Directions` rather than use `manageOrdersDirections`?

Reply:
The main reason is that the label for the field is "For example, arrangements schedule or prohibitive steps" which appears to be specific to this order so a more generic name might not be a good idea in this case.

Will change the name of this field to "manageOrdersDirectionsForC43" to match the other naming suggestion

Reply:
@samuelzerozero ^